contracts/contracts/StarknetCoreContractStub.sol

```diff
@@ -7,13 +7,13 @@ contract StarknetCoreContractStub {
         */
     function stateRoot() external view returns (uint256) {
         return
-            3321080970052968843805320357911538392446265524314601848332135665594593955343;
+            2793869633745137896926484933765493995916819896677591814984705621839603171245;
```

Comment:
what the meaning of this value?

Reply:
There is a comment at the top of the file mentioning its only to be used in tests. I have moved it into mocks folder to make it more explicit. Its mock contract used only unit tests